routing/loopback.go

```diff
@@ -0,0 +1,39 @@
+package routing
+
+import (
+	"context"
+	"net/http"
+)
+
+type (
+	routeChain struct {
```

Comment:
Would you like to limit number of routes in the loop?

From my side, it looks like there should be such a limit not to let this routeChain OOM-kill skipper pod.

Reply:
Yeah, maybe we can re-purpose existing max loopbacks limit config for this.